zcutil/build.sh

```diff
@@ -40,6 +40,14 @@ if [ -z "${CONFIGURE_FLAGS-}" ]; then
     CONFIGURE_FLAGS=""
 fi
 
+case "$@" in
+    *"V=1"*)
+        ;;
+    *)
+        CONFIGURE_FLAGS="--quiet $CONFIGURE_FLAGS"
+        ;;
+esac
```

Comment:
This matches unintended inputs like `xV=1x`. Also, I think that if `CONFIGURE_FLAGS` is set then it should be used as-is.

```suggestion
# Allow users to set arbitrary compile flags. Most users will not need this.
if [ -z "${CONFIGURE_FLAGS-}" ]; then
    # If the user did not set CONFIGURE_FLAGS, then use "--quiet" unless V=1 was given.
    CONFIGURE_FLAGS="--quiet"
    for arg in "$@"
    do
        if [ "$arg" = "V=1" ]; then
            CONFIGURE_FLAGS=""
        fi
    done
fi
```